packages/editor-ui/src/Interface.ts

```diff
@@ -22,6 +22,12 @@ import {
 	IWorkflowSettings as IWorkflowSettingsWorkflow,
 	WorkflowExecuteMode,
 } from 'n8n-workflow';
+import {
+	IRole,
+	IUser,
```

Comment:
I understand the why here... this is when I most regretted implementing design system separately from editor. let's discuss where these types should live

Reply:
Moved response types back to editor-ui.